java/ql/lib/ext/generated/org.apache.commons.io.model.yml

```diff
@@ -0,0 +1,1433 @@
+ 
+# THIS FILE IS AN AUTO-GENERATED MODELS AS DATA FILE. DO NOT EDIT.
+# Definitions of taint steps in the org.apache.commons.io framework.
```

Comment:
Not entirely accurate - there are also e.g. sinks.

Reply:
Yes, you are right. I will fix this in one of the follow ups.